tests/beta/queuing/test_beta_measure.py

```diff
@@ -268,11 +265,11 @@ def test_state_shape_and_dtype(self, wires):
 
         @qnode(dev)
         def func():
-            return qstate(range(wires))
+            return state()
 
-        state = func()
-        assert state.shape == (1, 2 ** wires)
-        assert state.dtype == np.complex128
+        state_ev = func()
```

Comment:
How come `ev`? It's not an expectation value :thinking:

Reply:
`state_val` perhaps better?

Reply:
Fixed! I was going for "ev"aluated, but yeah `state_val` looks better